src/Shared/FrameworkLocationHelper.cs

```diff
@@ -60,8 +60,10 @@ internal static class FrameworkLocationHelper
         internal static readonly Version dotNetFrameworkVersion40 = new Version(4, 0);
         internal static readonly Version dotNetFrameworkVersion45 = new Version(4, 5);
         internal static readonly Version dotNetFrameworkVersion451 = new Version(4, 5, 1);
+        internal static readonly Version dotNetFrameworkVersion452 = new Version(4, 5, 2);
```

Comment:
@AndyGerlicher Can you pay special attention to this file? It conflicted with some of your SDK addition commits.